src/web/src/pages/CoursePage.vue

```diff
@@ -70,10 +62,25 @@ export default {
     this.ready = true;
   },
   computed: {
-    prereqTree() {
-      let treeData = {};
-      treeData.name = this.courseName;
-      return treeData;
+    transformed() {
+      let precoreqtext = this.courseObj.raw_precoreqs;
+      if (precoreqtext === null) {
+        return "This course has no prerequisites";
+      }
+      const regex = /([A-Z]){4}( )([0-9]){4}/g;
+      while (precoreqtext.search(regex) != -1) {
+        let index = precoreqtext.search(regex);
+        let beforetext = precoreqtext.slice(0, index);
+        let dept = precoreqtext.slice(index, index + 4);
+        let cn = precoreqtext
```

Comment:
lets rename `cn` to `course_number`, had to think for a sec to figure this one out